apps/example-orchestrator-ui/components/Subscriptions/Subscriptions.tsx

```diff
@@ -200,10 +216,9 @@ export const Subscriptions: FC<SubscriptionsProps> = ({
         });
 
     const handleCriteriaChange = (criteria: Criteria<Subscription>) => {
-        if (criteria.page) {
-            const {
-                page: { index, size },
-            } = criteria;
+        const { page } = criteria;
```

Comment:
No desctructuring in the parameter?

`const handleCriteriaChange = ({ page }: Criteria<Subscription>) =>`

Reply:
Yes, good one